src/less/navbar.less

```diff
@@ -17,7 +17,7 @@
   &:extend(.clearfix all);
 
   @media (min-width: @grid-float-breakpoint) {
-    border-radius: @navbar-border-radius;
+    .border-radius(@navbar-border-radius);
```

Comment:
FWIW our browser support matrix doesn't have any browsers that dont just support `border-radius`